frontend/src/components/Modal/ridermodal.module.css

```diff
@@ -34,9 +63,16 @@
   grid-row: 3;
 }
 
-.gridR4 {
-  grid-row: 4;
+.error {
+  color: #ff647c;
```

Comment:
Still confirming with designers here but this font/font size pairing is not accessible. If I don't hear back soon I can make this fix in accessibility-fixes but I may request you to change this color because it doesn't meet contrast ratio.

Reply:
Update: change the color to #EB0023

Reply:
Make sure you change the error color here.